test/fixtures/fulfillment_order.json

```diff
@@ -0,0 +1,41 @@
+{
+   "fulfillment_order": {
+      "id": 519788021,
+      "shop_id": 690933842,
+      "order_id": 450789469,
+      "fulfillment_service_handle": "shipwire",
+      "status": "open",
+      "requires_shipping": true,
+      "is_not_deleted": true,
+      "deleted_at": null,
+      "created_at": "2016-07-12T11:23:42-04:00",
+      "updated_at": "2016-07-12T11:23:42-04:00",
+      "happened_at": "2016-07-12T11:23:42-04:00",
+      "assigned_location_id": 905684977,
+      "request_status": "unsubmitted",
+      "delivery_category": null,
+      "fulfillment_order_line_items": [
+         {
+            "id": 519788021,
+            "shop_id": 690933842,
+            "fulfillment_order_id": 519788021,
+            "line_item_id": 466157049,
+            "variant_id":  null,
+            "inventory_item_id":  null,
+            "inventory_commitment_id":  null,
```

Comment:
I think it would make sense for these (`variant_id`, `inventory_item_id`, `inventory_commitment_id`) to be set. In the real system, we shouldn't have `null` values here.

Reply:
Ok, I'll find and add some values.